### PR TITLE
Fix/webspace upload error

### PR DESCRIPTION
**Pull request by stentrop**
@plentymarkets/team-terra
@plentymarkets/ceres-io 
@fmutschler 

### Definition of Done
Documentation
- [x] Changelog (optional: relevant for every change which influences the API)
~~- [ ] Example (updated or created)~~
- [x] JSDoc (optional: relevant for every change which influences the API)

Testing
~~- [ ] Unit Test (updated or created)~~
- [x] Person 1 (mandatory)
- [ ] Person 2 (mandatory)

Browser-Support (relevant for changes in scss / appearance of components and jQuery)
- [x] Chrome
- [x] Firefox
- [x] Safari

### CODE REVIEW

File: src/assets/lang/locale-en.json [hunk 1]
```diff
@@ -9,6 +9,9 @@
 		"selectAll": "All"
 	},
 	"terraFileBrowser": {
+		"error": {
+			"tooLargePayload": "Datei zu groß"
```
Comment (dantrau) [inline src/assets/lang/locale-en.json]: translation

Comment (maxiroellplenty) [reply src/assets/lang/locale-en.json]: Lint error is fixed, translation @fmutschler

File: src/assets/lang/locale-de.json [hunk 2]
```diff
@@ -9,6 +9,9 @@
 		"selectAll": "Alle"
 	},
 	"terraFileBrowser": {
+		"error": {
+			"tooLargePayload": "Datei zu groß"
```
Comment (fmutschler) [inline src/assets/lang/locale-de.json]:
```suggestion
			"tooLargePayload": "Diese Datei ist zu groß."
```

File: src/assets/lang/locale-en.json (hunk 1 above)
Comment (fmutschler) [inline src/assets/lang/locale-en.json]:
```suggestion
			"tooLargePayload": "This file is too large to upload."
```

**Pull request merged**